style.css

```diff
@@ -106,12 +107,13 @@ img {
 
 /* Text */
 
-h1 {
+.main-headings {
 	font-family: var(--main-heading-font);
 	font-size: var(--font-xxl);
 }
 
-h2 {
+.sub-headings {
```

Comment:
You can probably target `h2` instead of adding this class to each one.

Reply:
done